lib/components/Keyboard/KeyboardInput/CustomKeyboardViewBase.js

```diff
@@ -6,11 +6,13 @@ export default class CustomKeyboardViewBase extends Component {
   static propTypes = {
     initialProps: PropTypes.object,
     component: PropTypes.string,
-    onItemSelected: PropTypes.func
+    onItemSelected: PropTypes.func,
+    useSafeArea: PropTypes.bool
```

Comment:
This is shared with Android, it should not be here (and the default should be in ios)